include/titan/prioritized_cache.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include "rocksdb/cache.h"
+
+namespace rocksdb {
+namespace titandb {
+
+class PrioritizedCache : Cache {
```

Comment:
s/`Cache`/`public Cache`/

and implement all other methods in `Cache` interface, by passing arguments directly to underlying `cache_`.

Reply:
ok, I have updated again now, please review again @yiwu-arbug

Reply:
I think you need to implement all methods (`Lookup`, `Release`, `Erase`, etc) in the `Cache` interface.